core/src/main/java/hivemall/math/matrix/sparse/DoKMatrix.java

```diff
@@ -309,6 +309,20 @@ public void eachNonZeroInColumn(@Nonnegative final int col,
         }
     }
 
+    public void eachNonZeroCell(@Nonnull final VectorProcedure procedure) {
+        if (nnz == 0) {
+            return;
+        }
+        final IMapIterator itor = elements.entries();
+        while (itor.next() != -1) {
+            long k = itor.getKey();
+            int row = Primitives.getHigh(k);
+            int col = Primitives.getLow(k);
+            double value = itor.getValue();
+            procedure.apply(row, col, value);
```

Comment:
`VectorProcedure#apply(@Nonnegative int row, @Nonnegative int col, double value)` should also be introduced.

Reply:
Oop, Thanks!